### PR TITLE
fix: Wrong validation of polkadot address at transfer? 

### LINKED ISSUE #7405: Wrong validation of polkadot address at transfer?
Seems does exist by subscan

https://polkadot.subscan.io/account/3p9ErbP5rmDnzu7PU7YFzJ6ih1J3DNE248rEJdkYLFShP6FY

image

### PR DESCRIPTION
**Pull request by hassnian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #7405

Test: address 3p9ErbP5rmDnzu7PU7YFzJ6ih1J3DNE248rEJdkYLFShP6FY

https://polkadot.subscan.io/tools/format_transform?input=3p9ErbP5rmDnzu7PU7YFzJ6ih1J3DNE248rEJdkYLFShP6FY&type=All

it's a Subsocial address 

CleanShot 2023-09-29 at 07 54 14@2x

valid polkadot ecosystem address but not one that we support atm

now you can transform from any valid polakdot ecosystem address to the current network

using this pr from Subsocial to Kusama returns -> Dg68oJbsCUtwNbKvGfeAsBftPaPnbrV9gSpJs2iJ1z24iJz 

CleanShot 2023-09-29 at 08 04 01@2x

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

CleanShot 2023-09-29 at 07 58 43

### <samp>🤖 Generated by Copilot at da86358</samp>

The code adds a feature to check the validity of Polkadot addresses and display an error message in the user's language if the address is from an unsupported network. The code modifies `AddressChecker.vue` and `locales/en.json` files.

### <samp>🤖 Generated by Copilot at da86358</samp>

> _Sing, O Muse, of the cunning coder who devised_
> _A way to check the Polkadot addresses and prevent_
> _The grievous error of sending funds to the wrong network_
> _By adding `transfers.invalidAddress.wrongNetwork` to the `locales/en.json` file._

### CODE REVIEW

File: components/shared/AddressChecker.vue [hunk 1]
```diff
@@ -106,6 +107,15 @@ const checkAddressByss58Format = (value: string, ss58: number) => {
   return isValid
 }
 
+const isValidUnsupportedPolkadotAddress = (address: string) => {
```
Comment (Jarsen136) [inline components/shared/AddressChecker.vue:110]: how about naming `isValidSupportedAddress` or `isValidAddress`

**Review by Jarsen136 — COMMENTED** [review]
Could you pls put the function into some utils file? 
otherwise lgtm

File: components/shared/AddressChecker.vue (hunk 1 above)
Comment (hassnian) [reply components/shared/AddressChecker.vue:110]: sure , as a general function isValidSupportedAddress or isValidAddress makes more sense

**Review by Jarsen136 — APPROVED** [review]

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 4.08 USD/DOT ~ 7.353 $DOT
🧗 [13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh](https://polkadot.subscan.io/account/13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)
🔗 [0x463b4b32b5f87072e5d005dbcd02329edffa253f0da93e39c1e8592e7887ae3e](https://polkadot.subscan.io/extrinsic/0x463b4b32b5f87072e5d005dbcd02329edffa253f0da93e39c1e8592e7887ae3e)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**